### PR TITLE
doc: doc for sql translation

### PR DESCRIPTION
**Pull request by raphaelvignes**
Small Doc about SQL VQB implementation

### CODE REVIEW

**Review by hachichaud — COMMENTED** [review]
Thanks for this doc!
I'm seeing this https://github.com/ToucanToco/weaverbird/blob/master/docs/_docs/contributing/new-step.md#adding-documentation in the contributing guidelines and wondering if the doc for the steps should be split into small files. What do you think?

**Comment by raphaelvignes** [comment]
[[Snowflake SQL VQB] - Aggregate/Group by Step Operations (3) [5]](https://trello.com/c/bwKVXdhY/1115-snowflake-sql-vqb-aggregate-group-by-step-operations-3-5)

**Comment by raphaelvignes** [comment]
I'll move the "steps" content in the relevant existing files and let the implementation logic in it's own file.

File: docs/_docs/tech/sql-steps/aggregate.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# `aggregate` step
```
Comment (hachichaud) [inline docs/_docs/tech/sql-steps/aggregate.md:1]: est-ce que tu pourrais juste ajouter ds cette step le fait que l'option keep granularity n'est pas supportée ? (enfin il me semble que cest ce quon s'était dit en planning)

Comment (raphaelvignes) [reply docs/_docs/tech/sql-steps/aggregate.md:1]: exact

**Review by hachichaud — COMMENTED** [review]
👍 juste une question

File: docs/_docs/tech/sql-steps/aggregate.md (hunk 1 above)
Comment (raphaelvignes) [reply docs/_docs/tech/sql-steps/aggregate.md:1]: fait

**Review by hachichaud — APPROVED** [review]

**Pull request merged**